### PR TITLE
CMQ-1439 Header ordering and change to use numbering

### PR DESCRIPTION
**Pull request by edjannoo**

### CODE REVIEW

File: source/connection-method/batch-process-direct/index.html.md.erb [hunk 1]
```diff
@@ -17,36 +17,18 @@ Make sure you only select this connection method when your users do not initiate
...
+  <%= partial 'connection-method/partials/gov-client-local-ips', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-client-local-ips-timestamp', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-client-mac-addresses' %>
+  <%= partial 'connection-method/partials/gov-client-timezone' %>
+  <%= partial 'connection-method/partials/gov-client-user-agent', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-client-user-ids', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-vendor-license-ids' %>
+  <%= partial 'connection-method/partials/gov-vendor-product-name' %>
+  <%= partial 'connection-method/partials/gov-vendor-version', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' }%>
+</ol>
```
Comment (czeczotka) [inline source/connection-method/batch-process-direct/index.html.md.erb:34]: I wonder if there is a Ruby way to give the local parameters in one statement for all partials?

Comment (edjannoo) [reply source/connection-method/batch-process-direct/index.html.md.erb:34]: I've been taking a look at some Ruby docs and can't see a way of doing that unfortunately

File: source/connection-method/batch-process-direct/index.html.md.erb [hunk 2]
```diff
@@ -17,36 +17,18 @@ Make sure you only select this connection method when your users do not initiate
...
-
-<%= partial 'connection-method/partials/vendor-version', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' }%>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/gov-vendor-license-id' %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/gov-vendor-product-name' %>
-<hr class="govuk-section-break govuk-section-break--visible">
+### Ed
```
Comment (czeczotka) [inline source/connection-method/batch-process-direct/index.html.md.erb:20]: 👋

File: source/connection-method/partials/gov-client-browser-js-user-agent.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<li>
+  <h3 id="gov-client-browser-js-user-agent">Gov-Client-Browser-JS-User-Agent</h3>
+
+  <p>JavaScript-reported user agent string from the <a href="../#originating-device">originating device</a>.</p>
+  <p>
+    Pass the value as reported by the browser, usually in this format:<br>
+    <code>&lt;product&gt; &sol; &lt;product-version&gt; &lpar;&lt;system-information&gt;&rpar; &lt;platform&gt; &lpar;&lt;platform-details&gt;&rpar; &lt;extensions&gt;</code>
```
Comment (czeczotka) [inline source/connection-method/partials/gov-client-browser-js-user-agent.erb:7]: Is there a reason for not using `<pre>`?

File: source/connection-method/partials/gov-client-timezone.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<li>
+  <h3 id="gov-client-timezone">Gov-Client-Timezone</h3>
+
+  <p>The local timezone of the <a href="../#originating-device">originating device</a>, expressed as <code>UTC±&lt;hh>:&lt;mm></code></p>
```
Comment (czeczotka) [inline source/connection-method/partials/gov-client-timezone.erb:4]: Is there a reason for not using <pre>?

File: source/connection-method/partials/gov-client-user-ids.erb [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+  <pre><code>Gov-Client-User-Ids: os=user123</code></pre>
+  <% end %>
+
+  <% if locals[:connection_method] == 'OTHER_VIA_SERVER' %>
+  <p><strong>Example</strong></p>
+
+  <pre><code>Gov-Client-User-Ids: os=user123&my-application=alice_account_id</code></pre>
+  <% end %>
+</li>
```
Comment (czeczotka) [inline source/connection-method/partials/gov-client-user-ids.erb:78]: You may need to rebase on top of my changes here.

**Review by czeczotka — APPROVED** [review]

**Pull request merged**